### ISSUE TITLE
[Functional] No action will perform while press 'Alt+ Enter' key on Video clip.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1367]
**Actual Result:**
No action will perform while press 'Alt+ Enter' key on Video clip.

**Note** - Same issue repro at all areas where video clip are present.

**Expected Result:**
Action should be perform while press 'Alt+ Enter' key on Video clip.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Navigate to 'Type your message' text box present in bottom of screen, type 'help' and press enter key.
3. Navigate to "Video card" button by using Tab key and press Enter on it to activate it.
4. Video will appear on the bottom of the screen, Navigate to 'Full Screen' button by using Tab key and press "Alt+Enter" for Full screen.
5. Observe that No action will perform while press 'Alt+ Enter' key 

**User Impact:**
If no action will perform while press 'Alt+ Enter' key on Video clip. user who is dependent on shortcut keys might get confuse about the range of that controller.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17763.107
**OS Version:** 1809
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[Functional Issue.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2588746/Functional.Issue.zip)

**Comment by compulim** [comment]
- [ ] Investigates if full-screening a standard HTML5 `<video>` element is doable

**Comment by corinagum** [comment]
This functionality is up to the browser and not in Web Chat's control.
